packages/core/src/types/select.d.ts

```diff
@@ -0,0 +1,14 @@
+export interface ISelectOption {
+  label: string;
+  value: string | number;
+  selected?: boolean;
+  highlighted?: boolean;
+}
+
+export interface ISelectConfig {
```

Comment:
I would rename this to `WayAutosuggestConfig`. The `I` is not a TypeScript convention.